v2/tools/generate-techniques-documentation.go

```diff
@@ -2,27 +2,28 @@ package main
 
 import (
 	"bytes"
-	"fmt"
-	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
-	_ "github.com/datadog/stratus-red-team/v2/pkg/stratus/loader"
-	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
 	"log"
 	"os"
 	"path/filepath"
 	"strings"
 	"text/template"
+
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	_ "github.com/datadog/stratus-red-team/v2/pkg/stratus/loader"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
 )
 
-func main() {
-	if len(os.Args) != 2 {
-		fmt.Fprintln(os.Stderr, "specify the docs output directory")
-		os.Exit(1)
+func GenerateTechDocs(docsDirectory string, techniques []*stratus.AttackTechnique, index map[stratus.Platform]map[string][]*stratus.AttackTechnique) error {
+	techniqueTemplate, err := os.ReadFile("tools/doc.tpl")
+	if err != nil {
```

Comment:
Handling this error, because otherwise `techniqueTemplate` is `""` and docs are generated with empty string. That threw me off for quite a bit 😅